chain/lf3/f3.go

```diff
@@ -0,0 +1,159 @@
+package lf3
+
+import (
+	"context"
+	"errors"
+	"time"
+
+	"github.com/ipfs/go-datastore"
+	"github.com/ipfs/go-datastore/namespace"
+	logging "github.com/ipfs/go-log/v2"
+	pubsub "github.com/libp2p/go-libp2p-pubsub"
+	"github.com/libp2p/go-libp2p/core/host"
+	"go.uber.org/fx"
+	"golang.org/x/xerrors"
+
+	"github.com/filecoin-project/go-f3"
+	"github.com/filecoin-project/go-f3/blssig"
+	"github.com/filecoin-project/go-f3/gpbft"
+
+	"github.com/filecoin-project/lotus/api"
+	"github.com/filecoin-project/lotus/build"
+	"github.com/filecoin-project/lotus/chain/stmgr"
+	"github.com/filecoin-project/lotus/chain/store"
+	"github.com/filecoin-project/lotus/node/modules/dtypes"
+	"github.com/filecoin-project/lotus/node/modules/helpers"
+)
+
+type F3 struct {
+	Inner *f3.F3
```

Comment:
```suggestion
	*f3.F3
```

Reply:
Unless there's a reason we don't want to expose all of F3.

Reply:
I think we shouldn't expose it at all unless there is an explicit reason. Looks like only GetCert and GetLatestCert is used by node impl. I'll export receivers for those explicitly.